### ISSUE TITLE
Riesgos python lib?

### ISSUE THREAD

**Issue by nbrinckm** [HugoRosero/System_Reliability#3]
We share some code here with the deus (reading the fragility functions and reading shakemaps). We should extract this code into an own package. This way we can support the same intensity files and fragility files in both deus and the system reliability service (and it should make maintainance easier too).

**Comment by HugoRosero** [comment]
I agree with your remark. In such way that package can also take over the job of processing intensity and fragility files of different nature (ASCII, scattered shapefiles, multi-intensity fragility functions, etc). In addition, that might solve the problem of which service is responsible for trasforming files from ASCII to TIFF in the case of the lahars, or even the package might be able to read ASCII without converting to TIFF (of course if this is an easier way thank making the transformation), and output the probabilities of failure for the System reliabiity WPS.
